examples/lfs/pre-commit

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
This won't actually fail the commit. Did you want to add `exit 1` in appropriate places?

Reply:
I do that.